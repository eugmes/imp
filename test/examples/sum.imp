function sum(x, y : integer) return integer is
var s : integer;
begin
    s = x + y;
    return s;
end sum;

procedure main () is
var a, b : integer;
begin
    output("Enter 2 integers: ");
    input(a); input(b);
    output("The sum is: ");
    output(sum(a, b));
    newline;
end main;
