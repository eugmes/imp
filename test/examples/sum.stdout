Enter 2 integers: The sum is: 113
